spec/helpers/configuration_helpers.rb

```diff
@@ -55,4 +53,29 @@ def self.dead_letter_configuration
       }]
     }
   end
+
+  # Contrary to subscriptions, delay exchanges are exchange based
+  def self.delay_exchange_configuration(exchange_name: 'name.delay', queue_name: 'name.delay', ttl: 30_000)
+    {
+      'exchange' => { 'name' => exchange_name, 'options' => { type: 'fanout', durable: true } },
+      'bindings' => [{
+        'queue' => { 'name' => queue_name, 'options' => {
+          durable: true, arguments: {
+            'x-dead-letter-exchange' => '', 'x-message-ttl' => ttl, 'x-dead-letter-routing-key' => 'queue_name'
```

Comment:
Not sure?
```suggestion
            'x-dead-letter-exchange' => '', 'x-message-ttl' => ttl, 'x-dead-letter-routing-key' => queue_name
```

Reply:
Not in this case, here queue name corresponds to the original queue. Might be worth turning that into a parameter though, if only for the purposes of documentation